pkg/proxy/userspace/proxier.go

```diff
@@ -1049,3 +1090,32 @@ func isClosedError(err error) bool {
 	// TODO: maybe create a stoppable TCP listener that returns a StoppedError
 	return strings.HasSuffix(err.Error(), "use of closed network connection")
 }
+
+const noConnectionToDelete = "0 flow entries have been deleted"
+
+// deleteServiceConnection use conntrack-tool to delete UDP connection specified by service ip
+func (proxier *Proxier) deleteServiceConnections(svcIPs []string) {
+	for _, ip := range svcIPs {
+		glog.V(2).Infof("Deleting connection tracking state for service IP %s", ip)
+		err := proxier.execConntrackTool("-D", "--orig-dst", ip, "-p", "udp")
+		if err != nil && !strings.Contains(err.Error(), noConnectionToDelete) {
```

Comment:
Yes, this was a direct port from pkg/proxy/iptables/proxier.go.  I forgot that we moved it over, so now we are changing kubernetes we could make the function common and just call it from both places.